digraph "unitGraph" {
    "r6 := @parameter0: java.lang.String[]"
    "$r0 = new compiler.escapeAnalysis.cr6716441.Tester"
    "specialinvoke $r0.<compiler.escapeAnalysis.cr6716441.Tester: void <init>()>()"
    "r1 = $r0"
    "specialinvoke r1.<compiler.escapeAnalysis.cr6716441.Tester: void test()>()"
    "goto [?= $r2 = <java.lang.System: java.io.PrintStream out>]"
    "$r5 := @caughtexception"
    "$r2 = <java.lang.System: java.io.PrintStream out>"
    "virtualinvoke $r2.<java.io.PrintStream: void println(java.lang.Object)>(r1)"
    "goto [?= return]"
    "$r4 := @caughtexception"
    "$r3 := @caughtexception"
    "return"
    "r6 := @parameter0: java.lang.String[]"->"$r0 = new compiler.escapeAnalysis.cr6716441.Tester";
    "r6 := @parameter0: java.lang.String[]"->"$r3 := @caughtexception";
    "$r0 = new compiler.escapeAnalysis.cr6716441.Tester"->"specialinvoke $r0.<compiler.escapeAnalysis.cr6716441.Tester: void <init>()>()";
    "$r0 = new compiler.escapeAnalysis.cr6716441.Tester"->"$r3 := @caughtexception";
    "specialinvoke $r0.<compiler.escapeAnalysis.cr6716441.Tester: void <init>()>()"->"r1 = $r0";
    "specialinvoke $r0.<compiler.escapeAnalysis.cr6716441.Tester: void <init>()>()"->"$r3 := @caughtexception";
    "r1 = $r0"->"$r5 := @caughtexception";
    "r1 = $r0"->"$r3 := @caughtexception";
    "r1 = $r0"->"specialinvoke r1.<compiler.escapeAnalysis.cr6716441.Tester: void test()>()";
    "specialinvoke r1.<compiler.escapeAnalysis.cr6716441.Tester: void test()>()"->"$r5 := @caughtexception";
    "specialinvoke r1.<compiler.escapeAnalysis.cr6716441.Tester: void test()>()"->"$r3 := @caughtexception";
    "specialinvoke r1.<compiler.escapeAnalysis.cr6716441.Tester: void test()>()"->"goto [?= $r2 = <java.lang.System: java.io.PrintStream out>]";
    "goto [?= $r2 = <java.lang.System: java.io.PrintStream out>]"->"$r4 := @caughtexception";
    "goto [?= $r2 = <java.lang.System: java.io.PrintStream out>]"->"$r3 := @caughtexception";
    "goto [?= $r2 = <java.lang.System: java.io.PrintStream out>]"->"$r2 = <java.lang.System: java.io.PrintStream out>";
    "$r5 := @caughtexception"->"$r4 := @caughtexception";
    "$r5 := @caughtexception"->"$r3 := @caughtexception";
    "$r5 := @caughtexception"->"$r2 = <java.lang.System: java.io.PrintStream out>";
    "$r2 = <java.lang.System: java.io.PrintStream out>"->"$r4 := @caughtexception";
    "$r2 = <java.lang.System: java.io.PrintStream out>"->"$r3 := @caughtexception";
    "$r2 = <java.lang.System: java.io.PrintStream out>"->"virtualinvoke $r2.<java.io.PrintStream: void println(java.lang.Object)>(r1)";
    "virtualinvoke $r2.<java.io.PrintStream: void println(java.lang.Object)>(r1)"->"$r4 := @caughtexception";
    "virtualinvoke $r2.<java.io.PrintStream: void println(java.lang.Object)>(r1)"->"$r3 := @caughtexception";
    "virtualinvoke $r2.<java.io.PrintStream: void println(java.lang.Object)>(r1)"->"goto [?= return]";
    "goto [?= return]"->"return";
    "goto [?= return]"->"$r3 := @caughtexception";
    "$r4 := @caughtexception"->"goto [?= return]";
    "$r4 := @caughtexception"->"$r3 := @caughtexception";
    "goto [?= return]"->"return";
    "$r3 := @caughtexception"->"return";
}
